Applications/Analyze/test/DoublePendulum3D_Setup_JointReaction_FrameKeyword.xml

```diff
@@ -0,0 +1,402 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<OpenSimDocument Version="20300">
+	<AnalyzeTool name="DoublePendulum3D_FrameKeyword">
+		<defaults>
```

Comment:
Minor, but you can reduce the size of these setup files by removing the mostly useless defaults block.